src/main/Java/SampleApp.java

```diff
@@ -0,0 +1,267 @@
+	}
+	
+	/**
+	 * Returns a valid integer from the console
+	 * @return integer representing user input
+	 * @throws IOException for incorrect user console input
+	 */
+	public Integer getValidUserInput() throws IOException{
+		Scanner scanner = new Scanner(System.in);
+		Integer inputInt = scanner.nextInt();
+		if(inputInt.equals(null)){
+			System.out.println("User input integer is null. Setting to 0 as default.");
+			inputInt = 0;
+		}
+		else if(inputInt == 9){
+			System.out.println("Exiting application...");
+			System.exit(0);
+		}
+		else if(inputInt != 0 && inputInt != 1 && inputInt != 2 && inputInt != 3 && inputInt != 4){
+			System.out.println("Invalid user input. Please enter an integer between 0 and 4.");
+			System.exit(0);
+		}
+		scanner.close();
+		return inputInt;
+	}
+	
+	/**
+	 * Uses HTTP POST to call /auth/authenticate and get an authenticated token to be passed for future API calls that require an authenticated user.
+	 * Uses the Properties from environment.properties to obtain username, password, and API-key
+	 * Parses json response and returns the value of the token from the json
+	 * @return String with the token
+	 * @throws IOException for REST HTTP POST
+	 */
+	public String getToken() throws IOException {
+		System.out.println("\nRunning example 1 to get token for authentication:");
+		System.out.println("Making REST HTTP POST call to " + props.getProperty("targetHost") + props.getProperty("authenticate") + 
+				" using API-key=" + props.getProperty("apiKey").substring(0, 8) + "...");
+		String tokenResponse = Request.Post(props.getProperty("targetHost") + props.getProperty("authenticate"))
+		.bodyString("{ \"userName\": \"" + props.getProperty("username") + "\","
+				+ " \"password\": \"" + props.getProperty("password") + "\" }", ContentType.APPLICATION_JSON)
+		.version(HttpVersion.HTTP_1_1)
+		.addHeader("Content-Type", "application/json")
+		.addHeader("API-key", props.getProperty("apiKey"))
+		.execute().returnContent().asString();
+		JsonObject jobj = new Gson().fromJson(tokenResponse, JsonObject.class);
+		System.out.println("Retrieved the following JSON from /auth/authenticate: \n" + jobj.toString());
+		String token = jobj.get("token").toString();
+		System.out.println("Retrieved token " + token);
+		return token;
+	}
+	
+	/**
+	 * Uses HTTP GET to obtain the JSON response as a string from the /search?searchTerm= API call
+	 * @throws IOException for REST HTTP GET
+	 * @return String with the searchResult
+	 */
+	public String getSearchResult() throws IOException{
+		System.out.println("\nRunning example 2 to search for certified products:");
+		System.out.println("Making REST HTTP GET call to " + props.getProperty("targetHost") + props.getProperty("search") + 
+				" using API-key=" + props.getProperty("apiKey").substring(0, 8) + "...");
+		String searchResult = Request.Get(props.getProperty("targetHost") + props.getProperty("search"))
```

Comment:
Not sure, but unless I'm misreading the ticket, that specific search is not specified in the requirements for ocd-887.